src/Forge.php

```diff
@@ -135,16 +135,19 @@ public function create()
     /**
      * Returns single server.
      *
-     * @param int $serverId
+     * @param int     $serverId
+     * @param boolean $reload   (optional) indicates whether the server should be reloaded
      *
      * @return \Laravel\Forge\Server
      */
-    public function get(int $serverId)
+    public function get(int $serverId, bool $reload = null)
```

Comment:
Ah yes, this was a hangover from when I erroneously had the parameter erroneously typed as `boolean` rather than `bool`. I kept getting the `PHP Fatal error:  Default value for parameters with a class type can only be NULL` error. My mistake.